### DISCUSSION TITLE
How to use firefly-iii on Arch Linux

### DISCUSSION THREAD

**Discussion by nandgator** [firefly-iii/firefly-iii#6042] [Firefly III] [answered]
This is funny and a bit embarrassing. I asked for a packaged version of `firefly-iii` @ https://github.com/chaotic-aur/packages/issues/1008.  

![image](https://user-images.githubusercontent.com/38415384/166153838-cd06b08f-7f0a-4396-9a21-b3aefb27f545.png)

Looks like they've added it ChaoticAUR, but after installing it, I cannot figure out how to launch a `firefly-iii` server. Any idea what's the launch command?

**Comment by JC5** [comment]
I have no idea, I didn’t package it. 😅

**Comment by nandgator** [comment, marked as answer]
As per https://aur.archlinux.org/packages/firefly-iii#comment-863468.

> This package is meant for use with a manually-configured webserver/PHP stack e.g. nginx with php-fpm. You can somewhat follow the instructions here: https://docs.firefly-iii.org/firefly-iii/installation/self_hosted/#configuration but you will have to adapt according to where the files are placed by this package.
